beest/sut/yards/eks-receiver/providers.tf

```diff
@@ -0,0 +1,31 @@
+terraform {
+  backend "s3" {
+    bucket = "lupulus-sandbox-terraform-state"
+    key    = "tiziano-aws-eks-1-21-shard.terraform.tfstate"
```

Comment:
Maybe use a more generic name for now

Reply:
agreed
also created this ticket https://stackstate.atlassian.net/browse/STAC-15553